integrated_channels/moodle/migrations/0023_alter_moodlelearnerdatatransmissionaudit_index_together.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 3.2.15 on 2023-01-03 14:48
+
+from django.db import connection, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('moodle', '0022_auto_20221220_1527'),
+    ]
+
+    db_engine = connection.settings_dict['ENGINE']
+    if 'sqlite3' in db_engine:
+        operations = [
+            migrations.AlterIndexTogether(
+                name='moodlelearnerdatatransmissionaudit',
+                index_together={('enterprise_customer_uuid', 'plugin_configuration_id')},
+            ),
+        ]
+    else:
+        operations = [
+            migrations.SeparateDatabaseAndState(
+                state_operations=[
+                    migrations.AlterIndexTogether(
+                        name='moodlelearnerdatatransmissionaudit',
+                        index_together={('enterprise_customer_uuid', 'plugin_configuration_id')},
+                    ),
+                ],
+                database_operations=[
+                    migrations.RunSQL(sql="""
+                        CREATE INDEX moodle_mldta_85936b55_idx
+                        ON moodle_moodlelearnerdatatransmissionaudit (enterprise_customer_uuid, plugin_configuration_id)
+                        ALGORITHM=INPLACE LOCK=NONE
+                    """, reverse_sql="""
+                        DROP INDEX moodle_dldta_85936b55_idx
```

Comment:
these two index names don't match between the forward and backward migrations, is that expected?

Reply:
good catch, thanks.